mlflow/tracking/fluent.py

```diff
@@ -1041,6 +1041,44 @@ def pagination_wrapper_func(number_to_get, next_page_token):
     return _paginate(pagination_wrapper_func, SEARCH_MAX_RESULTS_DEFAULT, max_results)
 
 
+def search_experiments(
+    view_type: int = ViewType.ACTIVE_ONLY,
+    max_results: Optional[int] = None,
+    filter_string: Optional[str] = None,
+    order_by: Optional[List[str]] = None,
+) -> List[Experiment]:
+    """
+    Search for experiments that match the specified search criteria.
+
+    :param view_type: Qualify requested type of experiments.
+    :param max_results: If passed, specifies the maximum number of experiments desired. If not
+                        passed, all experiments will be returned for the File and SQL backends.
+                        For the REST backend, the server will pick a maximum number of results
+                        to return.
+    :param filter_string: Filter query string, defaults to searching for all experiments.
+    :param order_by: List of column names with ASC|DESC annotation, to be used for ordering
+                     matching search results.
+    :return: A :py:class:`PagedList <mlflow.store.entities.PagedList>` of
+             :py:class:`Experiment <mlflow.entities.Experiment>` objects. The pagination token
+             for the next page can be obtained via the ``token`` attribute of the object.
```

Comment:
I don't think this return type is accurate - should just be a regular list of experiments.

Reply:
Thanks for the catch! It's not accurate. Fixed!